src/main/java/bhs/devilbotz/Robot.java

```diff
@@ -230,4 +233,12 @@ public static JsonNode getSysIdConstant(String name) {
   public static JsonNode getDriveTrainConstant(String name) {
```

Comment:
Javadocs, same with all other methods in Robot.json

Reply:
already had javadocs